tests/playwrightWebsite.test.ts

```diff
@@ -0,0 +1,59 @@
+import { test, expect } from '@playwright/test';
+import { NavigationMenuPage } from '../pages/NavigationMenuPage';
+import { PlaywrightDevPage } from '../pages/PlaywrightDevPage';
+import { DocumentationsPage } from '../pages/DocumentationsPage';
+import { FooterMenuPage } from '../pages/FooterMenuPage';
+
+test.describe("Playwright website:", () => {
+  let navigationMenu: NavigationMenuPage;
+  let playwrightDev: PlaywrightDevPage;
+  let documentations: DocumentationsPage;
+  let footerMenu: FooterMenuPage;
+
+  test.beforeEach(async ({ page }) => {
+    navigationMenu = new NavigationMenuPage(page);
+    playwrightDev = new PlaywrightDevPage(page);
+    documentations = new DocumentationsPage(page);
+    footerMenu = new FooterMenuPage(page);
+  });
+
+  test('should have correct title when select .Net in navigation menu', async () => {
+    playwrightDev.goto();
```

Comment:
`goto()`, `selectDotNetMenuItem()`, and all other methods which are `async` should be called with await: `await playwrightDev.goto()`